pkg/azure/client.go

```diff
@@ -31,6 +34,15 @@ type azureDNS struct {
 	}
 }
 
+// Helper function to convert string to *int32
+func toInt32Ptr(s string) *int32 {
+	i, err := strconv.Atoi(s)
+	if err != nil {
+		return nil
+	}
+	return to.Ptr(int32(i))
+}
+
```

Comment:
Can you move this helper function to a new file inside the same package? `pkg/azure/utils.go` I'd like to have those helpers isolated as it'll be easy for us to see what is re-used across providers.

Reply:
Sure!

Reply:
Done :)